run_tests.py

```diff
@@ -1,16 +1,22 @@
 import os
+import sys
 import subprocess
 
 
 def main():
     os.chdir(os.path.dirname(os.path.abspath(__file__)))
 
-    subprocess.call(['coverage', 'run', '-m', 'pytest', '-rw'])
+    if '--regtest' in sys.argv:
+        subprocess.call(['coverage', 'run', '-m', 'pytest', '--regtest', '-rw'])
+    else:
+        subprocess.call(
```

Comment:
If this errors, will subprocess.call raise an exception?

Reply:
In the future, we should probably do `subprocess.run(check=True)`

https://docs.python.org/3/library/subprocess.html#subprocess.run